WGLMakie/src/wglmakie.js

```diff
@@ -63,12 +63,14 @@ export function render_scene(scene, picking = false) {
 
 function start_renderloop(three_scene) {
     // extract the first scene for screen, which should be shared by all scenes!
-    const { fps } = three_scene.screen;
+    const { fps, renderer } = three_scene.screen;
     const time_per_frame = (1 / fps) * 1000; // default is 30 fps
     // make sure we immediately render the first frame and dont wait 30ms
     let last_time_stamp = performance.now();
     function renderloop(timestamp) {
         if (timestamp - last_time_stamp > time_per_frame) {
+            const canvas = renderer.domElement;
+            canvas.dispatchEvent( new Event('render') );
```

Comment:
I'm not sure if we should spam the websocket with a tick for every frame...
I know that this is the correct way, but maybe we need an approximation for WGLMakie for now?
We should definitely benchmark this, since an additional 30 messages per seconds could likely make message congestion worse.

Reply:
I moved it to the Julia side with a `Timer`. If I understand correctly these events wouldn't be synchronized well with rendering due to message passing anyway. The problem now is that Timer is just as bad as sleep in terms of accuracy, i.e. it's running at 20-25 ticks per second when targeting 30.

Reply:
I was thinking of generalizing #3954, merging it into this and using the same approach for WGLMakie ticks